core/arch/arm/mm/core_mmu_v7.c

```diff
@@ -642,8 +644,8 @@ void core_mmu_set_user_map(struct core_mmu_user_map *map)
 	} else {
 		write_ttbr0(read_ttbr1());
 	}
-	isb();
```

Comment:
I think this `isb()` is needed, but it should perhaps be moved to inside the braces above to make it clear that it belongs to `write_contextidr()` and `write_ttbr0()` respectively.